cli/config/credentials/native_store.go

```diff
@@ -4,6 +4,7 @@ import (
 	"github.com/docker/cli/cli/config/types"
 	"github.com/docker/docker-credential-helpers/client"
 	"github.com/docker/docker-credential-helpers/credentials"
+	"path/filepath"
```

Comment:
The linter is probably complaining about the import from go stdlib being in the wrong place, and wants it to be;

```go
import (
	"path/filepath"

	"github.com/docker/cli/cli/config/types"
	"github.com/docker/docker-credential-helpers/client"
	"github.com/docker/docker-credential-helpers/credentials"
)
```